lib/hanami/application/routing/resolver.rb

```diff
@@ -72,14 +77,20 @@ def register_slice_at_path(name, path)
         # @api private
         # @since 2.0.0
         def resolve_string_identifier(path, identifier)
-          slice_name = slice_registry.find(path) or raise "missing slice for #{path.inspect} (#{identifier.inspect})"
-          slice = slices[slice_name]
           endpoint_key = "#{ENDPOINT_KEY_NAMESPACE}.#{identifier}"
 
+          subject = if container.key?(endpoint_key)
+                      container
+                    elsif (slice_name = slice_registry.find(path))
+                      slices[slice_name]
```

Comment:
Should we have these the other way around? Otherwise if we had "actions.users.index" registered in both the app container as well as a slice, wouldn't it always find the app container version first?

Reply:
@timriley I'll add a specific test.

The behavior should give priority to app actions first and then slices'.